targets/targets.json

```diff
@@ -3198,7 +3198,7 @@
         "inherits": ["Target"],
         "core": "Cortex-M4F",
         "macros": ["TARGET_NRF52840", "BLE_STACK_SUPPORT_REQD", "SOFTDEVICE_PRESENT", "S140", "NRF_SD_BLE_API_VERSION=5", "NRF52840_XXAA", "NRF_DFU_SETTINGS_VERSION=1", "NRF_SD_BLE_API_VERSION=5", "CMSIS_VECTAB_VIRTUAL", "CMSIS_VECTAB_VIRTUAL_HEADER_FILE=\"cmsis_nvic.h\""],
-        "device_has": ["STCLK_OFF_DURING_SLEEP", "FLASH"],
```

Comment:
Was there political change of where `device has` should be?

Reply:
I asked around, and it seems like the popular opinion was that if the feature doesn't require specific pins on the board (like flash and TRNG) then it would be OK to put it in the MCU.

Reply:
ok